Add unique social title and description (#81)
stackblitz/docs: docs/platform/api/javascript-sdk-options.md

---
title: SDK Options Reference

> description: This page provides the available options for StackBlitz SDK projects.

Author: ❓ This page doesn't have an intro so I needed to make it.

Reviewer: ```suggestion
description: Modify your StackBlitz project with SDK options.
```
